censustree/censustree.go

```diff
@@ -50,8 +50,13 @@ type Options struct {
 	IndexAsKeysCensus bool
 }
 
-// TMP to be defined the production circuit nLevels
-const nLevels = 256
+// By default, the maximum number of levels will be 256, which allows to add
+// up to 2^256 leaves to the tree, with keys with up to 32 bytes. However the
+// number of levels could be setted up during the tree initialization which
+// allows to support uses cases with specific restrictions such as the
+// zkweighted census, which needs to optimize some artifacts size that depends
+// of the number of levels of the tree.
+const DefaultMaxLevels = 256
```

Comment:
I think we might use a default max levels up to 20 bytes.
On non zk censuses we currently support pubkeys (32 bytes) and addresses (20 bytes). We might skip the pubkey use case. We should create an issue to change this, so we dont forget.

Reply:
Done! https://github.com/vocdoni/vocdoni-node/issues/819